tools/windows/install-help/cal/PropertyReplacer.cpp

```diff
@@ -224,5 +224,15 @@ std::wstring replace_yaml_properties(
         }
     }
 
+    auto ec2UseWindowsPrefixDetection = propertyRetriever(L"EC2_USE_WINDOWS_PREFIX_DETECTION");
```

Comment:
What's the reason this property can't be replaced in the `for` loop above? It's not nested, does it have any other oddity I'm missing?

Reply:
Makes sense, thanks.